### PR TITLE
Add branding

### PR DESCRIPTION
**Pull request by b264**
No functional changes, only images and metadata

https://osraav.atlassian.net/browse/OSRA-460

### CODE REVIEW

**Comment by b264** [comment]
desktop

**Comment by b264** [comment]
safari_pinned_tab

**Comment by b264** [comment]
windows

**Comment by b264** [comment]
android_chrome

**Comment by b264** [comment]
ios

**Comment by Maroo-b** [comment]
@b264 that was amazing, just :shipit:  :)

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+		},
+		{
+			"src": "\/android-chrome-512x512.png?v=1000",
+			"sizes": "512x512",
+			"type": "image\/png"
+		}
+	],
+	"theme_color": "#86bc4d",
+	"display": "standalone"
+}
```
Comment (NikitaAvvakumov) [inline public/manifest.json]: While there is no standard for JSON, 8-space indentation just looks wrong :)

Comment (b264) [inline public/manifest.json]: ;-P

**Pull request merged**